cats_frontend/src/app/modules/cats/pages/cats-form/cats-form.component.ts

```diff
@@ -21,32 +21,27 @@ export class CatsFormComponent {
     });
 
     constructor(
-        private catsService: CatsService,
-        private messageService: MessageService,
         private router: Router,
-        private datePipe: DatePipe
+        private datePipe: DatePipe,
+        private store: Store,
     ) { }
 
-
-    submit() {
+    submit(): void {
         const cat: ICat = {
             name: this.form.get('name')?.value,
             breed: this.form.get('breed')?.value,
             birthday: this.datePipe.transform(this.form.get('birthday')?.value, 'YYYY-MM-dd')!,
             description: this.form.get('description')?.value,
-            comments: []
-          };
-          try {
```

Comment:
remove the try catch and the call the service to only dispatch an action for adding a cat, all the logic will be in the effect associated